java/src/jmri/jmrit/symbolicprog/CompositeVariableValue.java

```diff
@@ -493,10 +505,8 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
                 // some programming operation just finished
                 if (amReading) {
                     continueRead();
-                    return;
                 } else if (amWriting) {
                     continueWrite();
-                    return;
```

Comment:
Maybe these `return` should be restored as well?

Reply:
No, not at the end of a void method